oca_dependencies.txt

```diff
@@ -1 +1,2 @@
-partner-contact
\ No newline at end of file
+partner-contact
+hr https://github.com/initOS/hr.git 11.0-hr_holidays_public_generator
```

Comment:
The dependency should point to a OCA repository. If there is another PR needed for this it should be added to the PR description.

Reply:
The dependant PR was added in description. Shall I change the Line 
"hr https://github.com/initOS/hr.git 11.0-hr_holidays_public_generator" into "hr" now or after the dependant PR is merged?

Reply:
Before because this PR can only be merged when the other PR is done.

Reply:
But it looks poor if the tests are failed. No one will give a Review for it. Or do you mean that the Reviewer will not use the runbot to test it and just checkout all the necessary Branches locally and test it.